lib/vanagon/component/dsl.rb

```diff
@@ -184,7 +184,8 @@ def install_service(service_file, default_file = nil, service_name = @component.
           # Return here because there is no file to install, just a string read in
           return
         when "windows"
-          # Placeholder for Windows
+          @component.service = OpenStruct.new(:name => service_name, :service_file => service_file)
```

Comment:
@melissa were you just wondering what the call would look like? there are some examples in the mcollective component of PA, i.e. `pkg.install_service "ext/aio/redhat/mcollective.init", "ext/aio/redhat/mcollective.sysconfig", "mcollective"` and john didn't change anything about this function so it'll still look like that